### DISCUSSION TITLE
对最近节奏的一些问题

### DISCUSSION THREAD

**Discussion by AshStar-Y** [share121/inter-knot#2654] [其他]
最近的节奏中又搞出了什么，符华是崩3高管皮套人，有人说真有人说假，个人感觉是硬黑，不知道各位绳友怎么看

**Comment by fyhwsy** [comment]
鲁迅也干了

**Comment by alanyagger** [comment]
那个员工微博发的言论貌似是实锤确实逆天 但拿着什么皮套人去恶心角色厨纯纯nt  人物最初的灵感来源现实又不是多奇怪的事 非要把二次元跟现实一对一绑定

**Comment by shangjin2001** [comment]
评价为纯纯的没事找事，建议找个班上

**Comment by wwwsheepwww** [comment]
建议少接触不确定的节奏，真假参半的，不如等实锤再吃瓜

**Comment by demolute** [comment]
不过是些分不清现实和虚拟的小鬼在带节奏罢了

**Comment by cx525** [comment]
暑假作业还是少了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Nie7bai** [comment]
咱就一个打游戏的，少看节奏，多打游戏<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by blacktree-eme** [comment]
回应：低价收购相关周边
